src/status_im/ui/screens/communities/communities_list_redesign.cljs

```diff
@@ -0,0 +1,260 @@
+(ns status-im.ui.screens.communities.communities-list-redesign
+  (:require [re-frame.core :as re-frame]
+            [reagent.core :as reagent]
+            [status-im.i18n.i18n :as i18n]
+            [status-im.ui.components.list.views :as list]
+            [status-im.ui.components.react :as react]
+            [status-im.ui.screens.home.styles :as styles]
+            [quo2.components.community-card-view :as community-card-view]
+            [quo2.components.separator :as separator]
+            [quo2.components.text :as quo2.text]
+            [quo2.components.button :as quo2.button]
+            [quo2.components.counter :as quo2.counter]
+            [quo2.components.filter-tags :as filter-tags]
+            [quo2.components.filter-tag  :as filter-tag]
+            [quo2.foundations.colors :as quo2.colors]
+            [quo.components.safe-area :as safe-area]
+            [status-im.qr-scanner.core :as qr-scanner]
+            [quo2.components.tabs :as quo2.tabs]
+            [status-im.ui.screens.chat.photos :as photos]
+            [status-im.react-native.resources :as resources]
+            [status-im.multiaccounts.core :as multiaccounts]
+            [status-im.ui.components.topbar :as topbar]
+            [status-im.ui.components.plus-button :as plus-button]
+            [status-im.utils.handlers :refer [<sub]]
+            [quo2.components.icon :as icons])
+  (:require-macros [status-im.utils.views :as views]))
+
+(def selected-tab (reagent/atom :all))
+(def view-type   (reagent/atom  :card-view))
+(def sort-list-by (reagent/atom :name))
+
+(defn qr-scanner []
```

Comment:
i believe i've seen this component in other screens, if its a duplication can we move them in separate ns ?

Reply:
cool